### ISSUE TITLE
Creating item in O2M doesn't set ID

### ISSUE THREAD

**Issue by rijkvanzanten** [directus/api#1073]
Lets say I have a collection called `sections`. Creating a new card in the O2M field `cards` from `sections` item `15` like follows

```json
{
  "cards": [
    {
      "note": "Test card"
    }
  ]
}
```

should result in the item being saved in the database as

| id | note        | section |
|---|----------|--------|
| 1  | Test card | 15 |

**Comment by bjgajjar** [comment]
Hey @rijkvanzanten 

Maybe this is duplicate of https://github.com/directus/app/issues/1712 

Please check https://github.com/directus/app/issues/1712#issuecomment-496875714

**Comment by rijkvanzanten** [comment]
No, that particular issue was fixed in https://github.com/directus/app/pull/1804, but it doesn't take care of the problem listed above. It's a separate issue.

**Comment by bjgajjar** [comment]
@rijkvanzanten 

I take a pull from `o2m` branch of `APP` to replicate the issue. I was not able to `Select Existing` or `Add New` card from `APP`. There are some `vue` errors in the console.

(1) Error after `Add New`

![Screenshot_1](https://user-images.githubusercontent.com/19200178/61047801-e9701180-a3fd-11e9-9fa3-16eafd5fdac1.png)



(2) Error after `Select Existing`

![Screenshot_2](https://user-images.githubusercontent.com/19200178/61047825-fbea4b00-a3fd-11e9-9823-59b099029e81.png)

After clicking on `Select Existing` I was able to get popup to select the cards but when I submit it with `Done` it throws an error.


So I tried to check the API directly. Which works as expected. It seems that there is some issue from the `APP` in passing the payload.
